### ISSUE TITLE
read repositories recursively

### ISSUE THREAD

**Issue by kchang718** [vivekvjn/vscode-hg#2]
I have a source tree with subrepos, it would be nice if the extension picked up the repos underneath my thin repo containing the other leaf repos.

**Comment by Alexey-B** [comment]
Agree, this is a very necessary feature. Or maybe I can specify the desired paths in the project settings? Where is them to edit?
